CMakeLists.txt

```diff
@@ -144,7 +144,7 @@ if (NOT CMAKE_CXX_FLAGS)
   set(EXTRA_FLAGS "${EXTRA_FLAGS} -ftemplate-depth=512")
   # Reduce the number of template instantiations shown in backtrace to keep the
   # already insane error messages readable.
-  set(EXTRA_FLAGS "${EXTRA_FLAGS} -ftemplate-backtrace-limit=3")
+  set(EXTRA_FLAGS "${EXTRA_FLAGS} -ftemplate-backtrace-limit=0")
```

Comment:
Why not remove the flag entirely?

Reply:
the default is 10 on both gcc and clang.

Reply:
Ah, makes sense.